src/xdocs/style_configs.xml

```diff
@@ -21,8 +21,8 @@
       </p>
 
       <ul>
-        <li><a href="google_style.html">Google's style</a>;</li>
-        <li><a href="sun_style.html">Sun's style</a>.</li>
+        <li><a href="google_style.html">Google's style</a></li>
+        <li><a href="sun_style.html">Sun's style</a></li>
```

Comment:
it would be better to move this change to separate commit on this PR as fix is not related to issue.
you can use commit message prefix "minor: ......"
We usually use old commits as reference to amount of updates required for some periodical updates. So better to split unrelated changes in different commits.

Reply:
fixed, all updates pushed